.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,163 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  Core:
+    runs-on: ubuntu-latest
+    container: netmeld/netmeld-dev:latest
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure All
+      run: cmake -S . -B build
+
+    - name: Build Core
+      run: cmake --build build -j 2 --target netmeld-core
+
+    - name: Test Core
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-core
+        cd build
+        ctest -L ^Test.netmeld-core$
+    
+    - name: Package Core
+      run: |
+        cd ./scripts
+        bash ./make_netmeld-core_deb.sh
+        cd ..
+    
+    - name: Upload Deb Artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: netmeld-core-deb
+        path: scripts/*.deb
+  
+  
+  CoreModules:
+    needs: [Core]
+    
+    runs-on: ubuntu-latest
+    container: netmeld/netmeld-dev:latest
+
+    strategy:
+      matrix:
+        core-modules: [playbook-core, datalake-core]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure All
+      run: cmake -S . -B build
+
+    - name: Build Core Modules
+      run: cmake --build build -j 2 --target netmeld-${{ matrix.core-modules }}
+
+    - name: Test Core Modules
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-${{ matrix.core-modules }}
+        cd build
+        ctest -L ^Test.netmeld-${{ matrix.core-modules }}$
+    
+    - name: Package Core Modules
+      if: matrix.core-modules != 'datalake-core'
+      run: |
+        cd ./scripts
+        bash ./make_netmeld-${{ matrix.core-modules }}_deb.sh
+        cd ..
+    
+    - name: Upload Deb Artifact
+      if: matrix.core-modules != 'datalake-core'
+      uses: actions/upload-artifact@v2
+      with:
+        name: netmeld-${{ matrix.core-modules }}-deb
+        path: scripts/*.deb
+  
+  
+  Modules:
+    needs: [Core, CoreModules]
+    
+    runs-on: ubuntu-latest
+    container: netmeld/netmeld-dev:latest
+
+    strategy:
+      matrix:
+        modules: [core-bin, playbook, datalake, clw, exporters, fetchers, graphers, importers, inserters]
+       
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure All
+      run: cmake -S . -B build
+
+    - name: Build Modules
+      run: cmake --build build -j 2 --target netmeld-${{ matrix.modules }}
+
+    - name: Test Modules
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-${{ matrix.modules }}
+        cd build
+        ctest -L ^Test.netmeld-${{ matrix.modules }}$
```

Comment:
Just want to double check that this works as expected.  It seems odd the first `$` is evaluated but the final `$` is not.  I guess they don't actually look for `$` but `${{`...

Reply:
This seems to be working as expected, as far as I can tell.